prometheus-exporters-formula/prometheus-exporters/files/apache-exporter-systemd.config

```diff
@@ -0,0 +1,3 @@
+[Service]
+EnvironmentFile=
```

Comment:
Is there a reason you want to keep this empty option in the config file?

Reply:
Yes, it is to set it empty in case the option is set elsewhere. I don't want to use the `EnvironmentFile=` anymore unless there is a clear need for hiding the value.